test/unit/revision_test.rb

```diff
@@ -202,4 +202,12 @@ class RevisionsTest < ActiveSupport::TestCase
     tag_names = associated_tags.map(&:name)
     assert_not tag_names.include?('1234')
   end
+
+  test 'should be able to convert * [ ] into a checkbox [x]' do
+    revision = nodes(:one).latest
+    revision.body = "* [x]"
+    assert_includes revision.render_body, "<ul>\n<li>[x]</li>\n</ul>\n\n"
```

Comment:
Ah ok so here we'd want to search `render_body` to see if it has an `<input type="checkbox"...` in it, indicating that the markdown code for checkboxes has been properly replaced with an actual HTML checkbox. Make sense?

Thanks, this is exciting!

Reply:
OH, i see - so this test, once added, will fail, which is OK -- it confirms that we aren't yet actually doing the checkbox conversion. THEN, you should be able to go into the `def render_body` method inside `revision.rb`, and make sure we're using the `checkbox` function there, and that it really is replacing the output before it appears on, say, a wiki page. 

You can probably test this locally manually too, by going to http://localhost:3000/about and trying to add some checkboxes!

Reply:
@jywarren thanks for your feedback,  I will update this soon and do as you suggest :)